mob.go

```diff
@@ -507,189 +491,6 @@ func executeCommandsInBackgroundProcess(commands ...string) (err error) {
-		if err != nil {
-			say.Error(fmt.Sprintf("break timer couldn't be started on your system (%s)", runtime.GOOS))
-			say.Error(err.Error())
-			exit(1)
-		}
-	}
-
-	say.Info("It's now " + currentTime() + ". " + fmt.Sprintf("%d min break timer ends at approx. %s", timeoutInMinutes, timeOfTimeout) + ". Happy collaborating! :)")
-}
-
-func getUserForMobTimer(userOverride string) string {
-	if userOverride == "" {
-		return gitUserName()
-	}
-	return userOverride
-}
-
-func toMinutes(timerInMinutes string) int {
-	timeoutInMinutes, _ := strconv.Atoi(timerInMinutes)
-	if timeoutInMinutes < 0 {
-		timeoutInMinutes = 0
-	}
-	return timeoutInMinutes
-}
-
-func httpPutTimer(timeoutInMinutes int, room string, user string, timerService string, disableSSLVerification bool) error {
-	putBody, _ := json.Marshal(map[string]interface{}{
-		"timer": timeoutInMinutes,
-		"user":  user,
-	})
-	return sendRequest(putBody, "PUT", timerService+room, disableSSLVerification)
-}
-
-func httpPutBreakTimer(timeoutInMinutes int, room string, user string, timerService string, disableSSLVerification bool) error {
-	putBody, _ := json.Marshal(map[string]interface{}{
-		"breaktimer": timeoutInMinutes,
-		"user":       user,
-	})
-	return sendRequest(putBody, "PUT", timerService+room, disableSSLVerification)
-}
-
-func sendRequest(requestBody []byte, requestMethod string, requestUrl string, disableSSLVerification bool) error {
-	say.Info(requestMethod + " " + requestUrl + " " + string(requestBody))
-
-	responseBody := bytes.NewBuffer(requestBody)
-	request, requestCreationError := http.NewRequest(requestMethod, requestUrl, responseBody)
-
-	httpClient := http.DefaultClient
-	if disableSSLVerification {
-		transCfg := &http.Transport{
-			TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
-		}
-		httpClient = &http.Client{Transport: transCfg}
-	}
-
-	if requestCreationError != nil {
-		return fmt.Errorf("failed to create the http request object: %w", requestCreationError)
-	}
-
-	request.Header.Set("Content-Type", "application/json")
-	response, responseErr := httpClient.Do(request)
-	if e, ok := responseErr.(*url.Error); ok {
-		switch e.Err.(type) {
-		case x509.UnknownAuthorityError:
-			say.Error("The timer.mob.sh SSL certificate is signed by an unknown authority!")
-			say.Fix("HINT: You can ignore that by adding MOB_TIMER_INSECURE=true to your configuration or environment.",
-				"echo MOB_TIMER_INSECURE=true >> ~/.mob")
-			return fmt.Errorf("failed, to amke the http request: %w", responseErr)
-
-		default:
-			return fmt.Errorf("failed to make the http request: %w", responseErr)
-
-		}
-	}
-
-	if responseErr != nil {
-		return fmt.Errorf("failed to make the http request: %w", responseErr)
-	}
-	defer response.Body.Close()
-	body, responseReadingErr := io.ReadAll(response.Body)
-	if responseReadingErr != nil {
-		return fmt.Errorf("failed to read the http response: %w", responseReadingErr)
-	}
-	if string(body) != "" {
-		say.Info(string(body))
-	}
-	return nil
-}
-
```

Comment:
this is really hard to review becaues it is just one commit that changes everything. I dont know whether it was just moved, or wether something changed there, too

Reply:
Oh yes got your point. Should have done this in two seperate commits. I think i will just do it again and split it in two commits.